rpc/block_svc_routines.c

```diff
@@ -1621,6 +1621,7 @@ glusterBlockBuildMinCaps(void *data, operations opt)
     if (cblk->rb_size) {
       minCaps[GB_CREATE_RING_BUFFER_CAP] = true;
     }
+    minCaps[GB_CREATE_LOAD_BALANCE_CAP] = true;
```

Comment:
Will this be backward compatible if we set this to true by default?

Reply:
If someone is trying to Create from an updated gluster-block node, create will fail if one of the other nodes are not having load-balancing capability. Do we want to allow create to successed with this case ?

Reply:
If there are gluster-blockds without load-balancing support in the cluster, we need to downgrade the create operation to be done without load-balancing and succeed the operation.